float64[3] obb_xyz